packages/api/amplify_api/lib/src/graphql/web_socket/blocs/web_socket_bloc.dart

```diff
@@ -206,9 +206,13 @@ class WebSocketBloc with AWSDebuggable, AmplifyLoggerMixin {
   Stream<WebSocketState> _connectionAck(
     ConnectionAckMessageEvent event,
   ) async* {
+    // Connection is already established
+    if (_currentState is ConnectedState) {
```

Comment:
spoke offline, sounds like we are looking to a way to prevent this condition and only do 1 connection init in this scenario so we don't have to ignore a message like this (would only get 1 connection ack).

Reply:
Yeah this would appear to be creating multiple AppSync connections